Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -4622,6 +4635,11 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"\"$(PLATFORM_DIR)/Developer/Library/Frameworks\"",
```

Comment:
Yeah I'd be very hesitant to manually change any of the build settings of the Pods xcodeproj file 🎱